### ISSUE TITLE
Home screen

### ISSUE THREAD

**Issue by shirshman** [jeffreychan1/ivocate-issues#5]
Would it be possible to have the user's organizations, coalitions and bills show up on the home screen? I think it would be a very impressive prototype feature and show the power of the platform.

**Comment by jeffreychan1** [comment]
I agree.  Right now the home screen is just a place holder.  I'll add some dashboard like features this week.
